axelrod/strategy_transformers.py

```diff
@@ -57,6 +60,14 @@ def __init__(self, *args, **kwargs):
             else:
                 self.name_prefix = name_prefix
 
+        def __reduce__(self):
```

Comment:
Can we add a docstring here? I get that we're rebuilding the class but if I didn't know what was going on I wouldn't know until I read through the file, and this is a fairly complicated part of the library so I'd like to err on the side of more documentation.